lib/rubocop/cop/mixin/check_assignment.rb

```diff
@@ -31,7 +31,7 @@ def extract_rhs(node)
           _scope, _lhs, rhs = *node
         elsif node.op_asgn_type?
           _lhs, _op, rhs = *node
-        elsif node.send_type?
+        elsif node.send_type? || node.csend_type?
```

Comment:
Can you add a test case for this change?
RSpec is still green if I remove this change.

I guess it supports `something.some_method = 1` style assignment.

Reply:
> RSpec is still green if I remove this change.

Sorry, and thank you!

Pushed  48415e2f14b2b2515d372265dfa76136edd4c238